### PR TITLE
Added events for settings change (duplication, deletion, update)

### PR DESCRIPTION
**Pull request by webmake**

### CODE REVIEW

File: src/Controller/SettingsController.php [hunk 1]
```diff
@@ -74,6 +80,10 @@ public function quickEditAction(Request $request, string $domainName, string $se
 
         $setting->setData($value);
         $this->settingsManager->update($setting);
+        $this->eventDispatcher->dispatch(
+            SettingsManagerEvents::UPDATE_SETTING,
+            new DomainSettingChangeEvent($domainName, $settingName)
```
[the remark this answers is not in the document]
Comment (webmake) [reply src/Controller/SettingsController.php]: Do you mean name just `ChangeEvent`?

File: src/Controller/SettingsController.php [hunk 2]
```diff
@@ -22,15 +25,18 @@ class SettingsController extends AbstractController
...
         SettingsManager $settingsManager,
         SettingsAccessControl $settingsAccessControl,
-        ValidatorInterface $validator
+        ValidatorInterface $validator,
+        EventDispatcherInterface $eventDispatcher
     ) {
         $this->settingsManager = $settingsManager;
         $this->settingsAccessControl = $settingsAccessControl;
         $this->validator = $validator;
+        $this->eventDispatcher = $eventDispatcher;
```
[the remark this answers is not in the document]
Comment (webmake) [reply src/Controller/SettingsController.php]: As I see there is no other working code apart Controller with `SettingsManager` update methods, so any different implementation using those methods, should implement and event dispathing :/

[the remark this answers is not in the document]
Comment (martiis) [reply src/Controller/SettingsController.php]: [SettingsManager](https://github.com/HelisLT/settings-manager-bundle/blob/aa76a6ef67f0f326979d56d1a01cb0dd445edb02/src/Settings/SettingsManager.php) ?  :)

File: src/Controller/SettingsController.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (martiis) [reply src/Controller/SettingsController.php]: `SettingChangeEvent`

**Pull request merged**